forwards_paper.tex

```diff
@@ -1020,23 +1029,38 @@ \section*{Discussion}
 
 
 %%%%%%%%%%%%%%
-% \section*{Methods}
+\section*{Methods}
 
 \krt{Are we doing a methods section?}
 
-\subsection{Reagents list for the C++ implementation}
+\subsection{Requirements for the \cpp{} implementation}
 
 \begin{enumerate}
     \item \fwdpy{} 0.13.a0
     \item \fwdpp{} 0.5.7
     \item GNU Scientific Library 1.16
     \item pybind11 2.2.1
     \item GCC 4.8.5
-    \item msprime master branch
+    \item \msprime{} master branch \jda{which commit hash or short version number
+		    from pip freeze or {\tt msprime.\_\_version\_\_}?}
```

Comment:
We should just put in version 0.5.0 here I think. This will be released soon and won't differ in any meaningful way from the version used for these experiments.